build_projects/dotnet-cli-build/PathUtilities.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.IO;
+using Microsoft.Build.Utilities;
+using Microsoft.Build.Framework;
+using Microsoft.DotNet.Cli.Build.Framework;
+using Newtonsoft.Json;
+
+namespace Microsoft.DotNet.Cli.Build
```

Comment:
I would say we might as well copy these methods into the MakeRelative Task itself. Do you foresee it being used anywhere else?